guides/common/modules/proc_preparing-for-disaster-recovery-with-two-active-project-servers.adoc

```diff
@@ -0,0 +1,50 @@
+[id="preparing-for-disaster-recovery-with-two-active-project-servers"]
+= Preparing for disaster recovery with two active {ProjectServer}s
+
+Create a second {ProjectServer} by restoring a backup of your first {ProjectServer}.
+Configure both servers to operate independently in their respective data centers, but ensure that their content, such as repositories or host configuration, does not drift apart over time.
+
+.Procedure
+. Back up your {ProjectServer}.
+For more information, see xref:backing-up-{project-context}-server-and-{smart-proxy-context}_admin[].
+. Restore the backup on a system that will serve as your other {ProjectServer}.
+For more information, see xref:restoring-{project-context}-server-or-{smart-proxy-context}-from-a-backup_admin[].
++
+[NOTE]
+====
+Each server must have a distinct hostname and IP address.
+This enables you to re-register hosts if one of the servers fails.
+====
+. Ensure that content on your servers remains consistent:
+* If you want both servers to manage content synchronization and content view creation, follow these guidelines to prevent content drift:
+** Regularly synchronize repositories on both servers.
+For more information, see {ContentManagementDocURL}[Synchronizing repositories] in _{ContentManagementDocTitle}_.
+You can use the following Ansible modules to automate repository synchronization: `{ansible-namespace}.repository_sync` and `{ansible-namespace}.sync_plan`.
+** Ensure that content views on both servers match.
+* If you want one server to manage content synchronization and content view creation, you can configure Inter-Server Synchronization (ISS) so that the other server synchronizes its content from the first server.
+For more information, see {ContentManagementDocURL}Synchronizing_Content_Between_Servers_content-management[Synchronizing content between {ProjectServer}s] in _{ContentManagementDocTitle}_.
+* If you want one server to manage only content view creation but not content synchronization, you can configure the other server or multiple other servers to import content views from the first server but synchronize content from repositories.
+//      |------------ Satellite for CVs only ---------------|
+//                /                               \
+//              /                                   \
+//            /                                       \
+// |--Satellite for Hosts --|     |--Satellite for hosts--|
+. Register hosts to your servers so that each server manages hosts in its respective data center.
+. Automate running the `{foreman-maintain} health check` command on both servers.
```

Comment:
Should we mention an Ansible module here too? Or is there another way to automate this?

Reply:
There is no Ansible for it - user should opt to just use whatever they have, so I'd leave this as is.

Reply:
Ok, no change.